src/pr-info.ts

```diff
@@ -180,6 +180,8 @@ export async function queryPRInfo(prNumber: number) {
         const prInfo = info.data.repository?.pullRequest;
         if (!prInfo) return info; // let `deriveStateForPR` handle the missing result
         if (prInfo.mergeable !== "UNKNOWN") return info;
+        const { nodes, totalCount } = prInfo.files!;
+        if (nodes!.length < totalCount) console.log(`  *** Note: ${totalCount - nodes!.length} were not seen by this query!`);
```

Comment:
This is stuff that I'm probably the only one that will see it (the bot logs are not easy to get to)...  I therefore prefer `console.error` for it, or probably `console.warn` is better.

Reply:
Done :heavy_check_mark: